game/scripts/vscripts/abilities/oaa_glaives_of_wisdom.lua

```diff
@@ -57,7 +57,7 @@ end
 function modifier_oaa_glaives_of_wisdom:OnAttackStart(keys)
   local parent = self:GetParent()
   local ability = self:GetAbility()
-  if keys.attacker == parent and (keys.gain == 0 or ability:GetAutoCastState()) and ability:IsOwnersManaEnough() and (not keys.target:IsMagicImmune() or parent:HasScepter()) then
+  if keys.attacker == parent and (keys.gain == 0 or ability:GetAutoCastState()) and ability:IsOwnersManaEnough() and keys.target.IsMagicImmune and (not keys.target:IsMagicImmune() or parent:HasScepter()) then
```

Comment:
Won't this only return true if the target is magic immune and silencer has scepter?